modules/ROOT/pages/index.adoc

```diff
@@ -86,6 +86,21 @@ This documentation is for reference purposes only and will not be updated.
   xref:1.0@partner-manager::index.adoc[v1.0 Archive]
 
 | Studio Doc Archives |
+  xref:7.15@studio::index.adoc[v7.15 Archive] +
+  xref:7.14@studio::index.adoc[v7.14 Archive] +
+  xref:7.13@studio::index.adoc[v7.13 Archive] +
+  xref:7.12@studio::index.adoc[v7.12 Archive] +
+  xref:7.11@studio::index.adoc[v7.11 Archive] +
+  xref:7.10@studio::index.adoc[v7.10 Archive] +
+  xref:7.9@studio::index.adoc[v7.9 Archive] +
+  xref:7.8@studio::index.adoc[v7.8 Archive] +
+  xref:7.7@studio::index.adoc[v7.7 Archive] +
+  xref:7.6@studio::index.adoc[v7.6 Archive] +
+  xref:7.5@studio::index.adoc[v7.5 Archive] +
+  xref:7.4@studio::index.adoc[v7.4 Archive] +
+  xref:7.3@studio::index.adoc[v7.3 Archive] +
+  xref:7.2@studio::index.adoc[v7.2 Archive] +
+  xref:7.1@studio::index.adoc[v7.1 Archive] +
   xref:5.x@tcat-server::index.adoc[v5 Archive]
```

Comment:
```suggestion
  xref:5.x@studio::index.adoc[v5 Archive]
```

Reply:
I'm not sure why that says tcat-server...it's what's currently there on archive! I must've messed up early on :>

Reply:
I don't know what's that, but as it is working I didn't know we had to fix it! hahahaha
Thanks!